docs/submit/install-ospool-ap.md

```diff
@@ -4,44 +4,46 @@ title: Installing an Open Science Pool Access Point
-If your local batch system frequently has many jobs waiting to run for a long time,
-you do not have a local batch system,
-or if you simply want to provide a local entry point for OSG-bound jobs,
-adding a path to OSG may result in less waiting for your users.
+This document explains how to add a path for user jobs to flow from your local site out to the Open Science Pool (OSPool),
+which in most cases means that the jobs will have far more capacity available to run on than locally.
+This is useful if:
+
+- your local batch system frequently has many jobs waiting to run for a long time
+- you do not have a local batch system
+- you want to provide a local entry point for OSPool-bound jobs
 
 Note that if you do not have a local batch system, consider having your users use
 [OSG Connect](https://portal.osg-htc.org/documentation), which will require less
 infrastructure work at your site.
 
 !!!note
-    Flocking to the OSG requires some modification to user workflows.
+    Flocking to OSPool resources requires some modification to user workflows.
     After installation, see the [usage](#usage) section for instructions on what your users will need to do.
 
 
 Background
 ----------
 Every batch computing system has one or more entry points that users log on to and use to hand over their computing work
 to the batch system for completion.
-For the HTCondor batch system, we say that users log on to a access point (i.e., submit node, submit host) to submit
-their jobs to HTCondor, where the jobs wait ("are queued") until computing resources are available to run them.
-In a purely local HTCondor system, there are one to a few access points and many computing resources.
+For the HTCondor batch system, we say that users log on to an Access Point (AP)
+(also known as a submit node or submit host)
+to submit their jobs to HTCondor, where the jobs wait ("are queued") until computing capacity is available to run them.
+In a purely local HTCondor system, there are one to a few Access Points and many computing resources.
 
-An HTCondor access point can also be configured to forward excess jobs to an OSG-managed pool.
+An HTCondor Access Point can also be configured to forward excess jobs to the Open Science Pool.
 This process is called [flocking](https://htcondor.readthedocs.io/en/latest/grid-computing/connecting-pools-with-flocking.html).
 If you already have an HTCondor pool, we recommend that you install this software
-on top of one of your existing HTCondor access points.
+on top of one of your existing HTCondor Access Points.
 This approach allows a user to submit locally and have their jobs run locally or,
-if the user chooses and if local resources are unavailable, have their jobs automatically flock to OSG.
+if the user chooses and if local capacity is unavailable, have their jobs automatically flock to the OSPool.
 If you do not have an HTCondor batch system, following these instructions will install the HTCondor submit service
```

Comment:
```suggestion
If you do not have an HTCondor batch system, following these instructions will install the HTCondor Access Point
```

Probably this instead?

Reply:
That's a grey area. I think of the AP as the entire machine (or VM or container); this sentence refers to installing the condor daemons instead.

Reply:
I think installing an HTCondor AP is where we want to go brand-wise: the HTCSS team will be delivering on shipping a `condor-ap` package sometime in the next month or so (same with the EP product)

Reply:
how about "HTCondor Access Point software" ?

Reply:
Works for me

Reply:
Done. I used the abbreviation because I defined it in the previous paragraph.